### ISSUE TITLE
Issue with contrast-agnostic soft SC predictions

### ISSUE THREAD

**Issue by naga-karthik** [ivadomed/canproco#72]
I was going through contrast-agnostic soft predictions on PSIR/STIR images along with @valosekj and found an issue with this line: 

https://github.com/ivadomed/canproco/blob/7316f4cff9bac0c72f2db0582b091b2b319e32ac/segment_sc_contrast-agnostic/01_segment_sc_contrast-agnostic.sh#L181C12-L181C12

While it seems like this line adds 4 predictions (the original + 3 flipped), in reality it is indeed adding only 2 predictions. The max value of the soft prediction is 2 (see the picture below)

<details>

<summary> incorrect max value </summary>

<img width="454" alt="Screen Shot 2023-12-12 at 6 40 01 PM" src="https://github.com/ivadomed/canproco/assets/53445351/ba58fa22-87d8-476a-8c9c-7fae9f48f3e9">

</details>

I corrected this line locally using `-add` only once and the images to add in a list:

```
sct_maths -i sub-cal056_ses-M0_STIR_flip_x_pred_back.nii.gz -add sub-cal056_ses-M0_STIR_flip_y_pred_back.nii.gz sub-cal056_ses-M0_STIR_flip_z_pred_back.nii.gz sub-cal056_ses-M0_STIR_pred.nii.gz -o sub-cal056_ses-M0_STIR_pred_sum4.nii.gz
```

and got the following result:

<details>

<summary> correct max value </summary>

<img width="454" alt="Screen Shot 2023-12-12 at 6 42 05 PM" src="https://github.com/ivadomed/canproco/assets/53445351/85426ce9-87e9-4968-9127-c89011a549b5">

</details>

This error should not have affected an immediate results as these predictions were binarized [here](https://github.com/ivadomed/canproco/blob/main/segment_sc_contrast-agnostic/01_segment_sc_contrast-agnostic.sh#L185). However, these soft preds _cannot_ be used for re-training the contrast-agnostic model with STIR/PSIR contrasts.

**Comment by valosekj** [comment]
Yeah, sorry, my mistake. I should have used only a single `-add`. I do not know why I used `-add` three times.
